feature_engine/wrappers/wrappers.py

```diff
@@ -125,3 +125,61 @@ def transform(self, X: pd.DataFrame) -> pd.DataFrame:
             X[self.variables] = self.transformer.transform(X[self.variables])
 
         return X
+
+
+    def inverse_transform(self, X: pd.DataFrame) -> pd.DataFrame:
+        """
+        Reverse the transformation of a dataframe.
+        Only the selected features will be modified.
+
+        Parameters
+        ----------
+        X: pandas dataframe of shape = [n_samples, n_features]
+            A dataframe that has already been transformed using a
+            sklearn transformer.
+
+        Returns:
+        X_new : pandas dataframe of shape = [n_samples, n_features]
+
+        """
+        # check that input is a dataframe
+        X = _is_dataframe(X)
+
+        # A OneHotEncoder-transformed dataframe has the original variables.
+        if isinstance(self.transformer, OneHotEncoder):
+            X = X[self.variables]
```

Comment:
self.variables may be a subset of the variables. We want to drop the features that are output of get_feature_names